README.md

```diff
@@ -52,10 +52,10 @@ or Wget:
 wget -qO- https://raw.githubusercontent.com/creationix/nvm/v0.33.11/install.sh | bash
 ```
 
-<sub>The script clones the nvm repository to `~/.nvm` and adds the source line to your profile (`~/.bash_profile`, `~/.zshrc`, `~/.profile`, or `~/.bashrc`).</sub>
+<sub>The script clones the nvm repository to `$XDG_CONFIG_HOME/nvm` and adds the source line to your profile (`~/.bash_profile`, `~/.zshrc`, `~/.profile`, or `~/.bashrc`).
```

Comment:
I'd still leave this as `~/.nvm`, but maybe clarify that it will go inside `$XDG_CONFIG_HOME` if present, otherwise will fall back to `$HOME`.

(also, you've lost the `</sub>` here)